lib/local-links-manager/import/local_authorities_importer.rb

```diff
@@ -94,7 +95,8 @@ def mapit_authorities
       end
 
       def mapit_service_response
-        Services.mapit.areas_for_type(self.class.local_authority_types)
+        mapit = GdsApi.mapit(disable_cache: Rails.env.test?)
```

Comment:
Part of the reason for using a services module is to allow easy swapping out of clients for mock or fake versions in tests.  Adding a direct call here means that less elegant stubbing (or request mocking) would be required to mock the result of this request.

Reply:
I don't think using `GdsApi` makes this any harder, you can do `allow(GdsApi).to receive(:mapit).and_return(my_fake_mapit_instance)`.

Reply:
It doesn't make it harder but it does make it more magic.  That's what I meant by "less elegant stubbing".

Reply:
I've changed it now so the `Services` class returns instances from `GdsApi` but the users continue to have `Service.` rather than access `GdsApi` directly.